src/test/java/io/confluent/examples/streams/SumLambdaIntegrationTest.java

```diff
@@ -89,33 +79,29 @@ public void shouldSumEvenNumbers() throws Exception {
         .reduce((v1, v2) -> v1 + v2);
     sumOfOddNumbers.toStream().to(outputTopic);
 
-    final KafkaStreams streams = new KafkaStreams(builder.build(), streamsConfiguration);
-    streams.start();
+    final TopologyTestDriver topologyTestDriver = new TopologyTestDriver(builder.build(), streamsConfiguration);
 
     //
     // Step 2: Produce some input data to the input topic.
     //
-    final Properties producerConfig = new Properties();
-    producerConfig.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, CLUSTER.bootstrapServers());
-    producerConfig.put(ProducerConfig.ACKS_CONFIG, "all");
-    producerConfig.put(ProducerConfig.RETRIES_CONFIG, 0);
-    producerConfig.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class);
-    producerConfig.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class);
-    IntegrationTestUtils.produceValuesSynchronously(inputTopic, inputValues, producerConfig);
+    IntegrationTestUtils.produceKeyValuesSynchronously(
+      inputTopic,
+      inputValues.stream().map(i -> new KeyValue<Void, Integer>(null, i)).collect(Collectors.toList()),
```

Comment:
if you drill into `IntegrationTestUtils.produceValuesSynchronously`, this is what we were sending before.